### PR TITLE
Added new User Functionality

### PR DESCRIPTION
**Pull request by anubhakushwaha**

### CODE REVIEW

**Comment by chhavip** [comment]
@anubhakushwaha , updates?

**Comment by anubhakushwaha** [comment]
CHANGES MADE:
1.Resetting the stored data in Avatar table of the previous user
2.Resetting values in Session History datamodel.
3.Added two methods:
 a)updateComplete : setting up the value of completed attribute of Scenario as zero(assures that no scene is completed when new user comes)
b)resetPurchase : setting the purchased attribute of tables - Clothes ,Hair and Accessory .

- All the changes are done in AvatarRoomActivity which  is opened  only when app installs for the first time and user selects Start or when new user is selected.
- The methods definition are added in DatabaseHandler class

**Comment by chhavip** [comment]
Great thanks! will review.

**Comment by anubhakushwaha** [comment]
@chhavip Missed one more attribute to be updated,done all the changes properly now in PR #168 ,please remove the tag from here and add it to the other one. Thanks !

File: PowerUp/app/src/main/java/powerup/systers/com/AvatarRoomActivity.java [hunk 1]
```diff
@@ -242,6 +242,19 @@ public void onClick(View v) {
                 getmDbHandler().setAvatarFace(face);
                 getmDbHandler().setAvatarHair(hair);
                 getmDbHandler().setAvatarCloth(cloth);
+                getmDbHandler().setAvatarBag(0);
+                getmDbHandler().setAvatarGlasses(0);
+                getmDbHandler().setAvatarHat(0);
+                getmDbHandler().setAvatarNecklace(0);
+                getmDbHandler().updateComplete();//set all the complete scenario fields back to 0
+<<<<<<< HEAD
```
Comment (chhavip) [inline PowerUp/app/src/main/java/powerup/systers/com/AvatarRoomActivity.java]: Please remove git merge comments as they would not let the code compile. Remove similar statements from the rest of the code too.

**Pull request closed without merging**